documentation/modules/serving/pages/knative-client.adoc

```diff
@@ -306,17 +305,15 @@ To delete a specific revision you can use the command:
 [.console-input]
 [source,bash,subs="+macros,attributes+"]
 ----
-kn revision delete greeter-7cqzq
+kn revision delete greeter-twpgf-1
 ----
 
 [.console-output]
 [source,bash]
 ----
-Revision 'greeter-7cqzq' successfully deleted in namespace 'knativetutorial'.
+Revision 'greeter-twpgf-1' successfully deleted in namespace 'knativetutorial'.
 ----
 
-Now <<kn-invoke-service, invoking service>> will return the response from revision `greeter-6m45j`.
-
```

Comment:
This statement is not true. When you delete the revision, a new revision is created with the same configuration. So, it will still return the same response as the deleted revision.